main.tf

```diff
@@ -37,8 +37,14 @@ terraform {
   }
 }
 
+locals {
+  gcp_zone_parts = split("-", var.gcp_zone)
```

Comment:
These locals were also added for L2 wiring and could be taken out

Reply:
You're saying to also remove line 47 then?